tests/templates/service/check_render.py

```diff
@@ -41,7 +41,7 @@ def check_file_template(self):
 
 class TemplateRenderingService(Service):
     NO_VARIABLE = "fixed content"
-    SIMPLE_VARIABLE = "Hello {{name}}!"
+    SIMPLE_VARIABLE = "Hello {{a_field}}!"
```

Comment:
On second look, having both `name` and `id` when they're conceptually very similar is odd. 
I think I will consolidate into one property called `service_id`